pyproject.toml

```diff
@@ -10,7 +10,17 @@ pandas = "^1.3.5"
 
 [tool.poetry.dev-dependencies]
 flake8 = "^5.0.4"
-isort = "^5.10.1"
+isort = "^5.11.4"
+pytest = "^7.2.1"
+coverage = "^7.0"
+pytest-cov = "^4.0.0"
+chispa = "*"
+pytest-dependency = "^0.5.1"
+pytest-tap = "^3.3"
+
+[tool.pytest.ini_options]
+junit_test_suite_name = "sml_small"
+addopts = "--exitfirst --tap-stream --cov=sml_small --cov-fail-under=90"
```

Comment:
May wish to remove --tapstream as already present in github workflow and will allow more flexibility in running tests.

Reply:
Removed =)